ChatQnA/tests/_test_chatqna_pinecone_on_xeon.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
In the new test structure, the test scripts name should be `ChatQnA/tests/test_compose_pinecone_on_xeon.sh`